examples/example_otap.py

```diff
@@ -38,6 +38,10 @@
                         help="Network address concerned by scratchpad")
     parser.add_argument('--file',
                         help="Scratcphad to use")
+    parser.add_argument("--gateway",
+                        type=list,
+                        help="Gateway list to use. If specified, the OTAP will be performed on given Gateway IDs",
```

Comment:
Should we support also input of the gateway list from a file ?

Reply:
It is supported by default as we are using parser API.
So any parameter can come from cmd line or a file. File is passed to the script by prefixing it by @
Example:
example_otap.py --host ... --username ... @list_of_agateways